src/app/app.module.ts

```diff
@@ -77,11 +77,11 @@ import { PortalStatisticsComponent } from './components/portal-statistics/portal
         redirectUrl: window.location.origin,
         postLogoutRedirectUri: window.location.origin,
         clientId: environment.identityServerClientId,
-        scope: 'openid profile email', // Adjust the scopes as needed
+        scope: 'openid profile email offline_access',
```

Comment:
This requires also some changes in the default realm configuration I think

Reply:
Ah, perhaps, I am not aware, but this scope is required for renewing tokens, according to the plugin. Let me review it further.

Reply:
https://keycloak-test.healthdata.nl/realms/ckan/.well-known/openid-configuration

If you check this url, the scope is avaiable, so perhaps nothing is required in Keycloak to be configured.